samples/CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ if (BUILD_ENCLAVES)
                     log_callback switchless
           DESTINATION ${CMAKE_INSTALL_DATADIR}/openenclave/samples)
 
+  if (UNIX)
```

Comment:
if SGX?

Reply:
This file seems to be SGX specific. It does not have any OPTEE vs SGX checks and exports SGX only samples like switchless, data-sealing, attestation, attested_tls etc.

Hence, omitting the SGX only check for apkman sample.